kythe/go/platform/vfs/vfs.go

```diff
@@ -54,7 +54,7 @@ type Reader interface {
 	Stat(ctx context.Context, path string) (os.FileInfo, error)
 
 	// Open opens an existing file for reading, as os.Open.
-	Open(ctx context.Context, path string) (io.ReadCloser, error)
+	Open(ctx context.Context, path string) (FileReader, error)
```

Comment:
As the test failures indicate, making this change is going to mean changing everything which currently implements this interface.

Reply:
Yeah I forgot about the zip fs implementation :/.